### PR TITLE
[sil_khmer]: update keyboard layout and documentation, and simplified the keyboard usage

### PR DESCRIPTION
**Pull request by Meng-Heng**
This PR should solve #2703 

There is only 1 replacement key and everything else should stay the same but I updated the document so it is both desktop and mobile friendly.

Should I update the help.keyman.com document too for this keyboard?
Thank you!

### CODE REVIEW

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by LornaSIL** [comment]
The .kpj file is incompatible with our current build system. Could you revert the changes made to the .kpj? It needs to be compatible with Keyman Developer 16.0 and include all the files.

**Comment by Meng-Heng** [comment]
Hi @LornaSIL, I created another PR because the changes are in a new branch.  #2718

**Comment by mcdurdin** [comment]
> The .kpj file is incompatible with our current build system.

Note: I am working on the 17.0 changes. Next week I will coordinate with you all on the merge, and then we will be able to accept 17.0 files. So we can probably accept these or just wait for the merge of 17.0 changes for these kinds of PRs.

**Comment by Meng-Heng** [comment]
Thank you, Marc. I will close the 2718 PR I have.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by LornaSIL** [comment]
In the .php file, for the touch layout, the syntax should be change from `osk` to `osk-tablet`:

```
<div id='osk-tablet' data-states='default shift'>
</div>
```
Otherwise, I think it will be okay once v17 is being used as our build agent.

**Comment by LornaSIL** [comment]
I'm not sure what the status of this PR is. Did another PR take the place of this one?

File: release/sil/sil_khmer/source/help/sil_khmer.php [hunk 1]
```diff
@@ -4,32 +4,133 @@
...
 
 <h2>Desktop Keyboard Layout</h2>
 <div id='osk' data-states='default shift alt ctrl shift-ctrl ctrl-alt shift-ctrl-alt'>
 </div>
 
-
-
+<h2>Touch Keyboard Layout</h2>
+<div id='osk' data-states='default shift'>
+</div>
```
Comment (Meng-Heng) [inline release/sil/sil_khmer/source/help/sil_khmer.php:136]:
```suggestion
<div id='osk-tablet' data-states='default shift'>
</div>
```

**Comment by Meng-Heng** [comment]
I applied your suggestion now, Lorna. This is the official PR, another one is abandoned.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Comment by keyman-server** [comment]
Thank you for your pull request. The Keyman keyboard review team have been notified of your pull request and will review it and build it shortly.

**Review by LornaSIL — APPROVED** [review]
LGTM (I had to change the .kpj to version 16 format because v17 doesn't compile yet).

**Comment by LornaSIL** [comment]
@Meng-Heng 
I'm sorry, but I didn't check this keyboard good enough. The version number in the .kmn was not bumped.
This statement needs changing to 2.0
```
store(&KEYBOARDVERSION) '1.5'
```
You'll need to submit a new PR because I didn't catch this issue. I think just changing the version number is sufficient so it's been changed everywhere else.

**Comment by Meng-Heng** [comment]
Me neither. No worries, I will get it done. Thank you @LornaSIL❤️

**Pull request merged**